### PR TITLE
test: Fix vet complaints about formatting directives

### PR DESCRIPTION
**Pull request by titanous**

### CODE REVIEW

**Comment by lmars** [comment]
LGTM

File: test/cluster/cluster.go [hunk 1]
```diff
@@ -495,7 +495,7 @@ func (c *Cluster) DumpLogs(w io.Writer) {
 		return err
 	}
 	fallback := func() {
-		fmt.Fprintln(w, "\n*** Error getting job logs via flynn-host, falling back to tail log dump\n")
+		fmt.Fprintf(w, "\n*** Error getting job logs via flynn-host, falling back to tail log dump\n\n")
```
Comment (heavenlyhash) [inline test/cluster/cluster.go]: ...does `vet` just dislike the `Fprintln` function in general?  There's no format directives.

Comment (titanous) [inline test/cluster/cluster.go]: It got confused by the trailing newline. Instead of leaving the warning their I got rid of it by changing the call.

**Comment by heavenlyhash** [comment]
LGTM

**Pull request merged**